compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -1488,6 +1488,17 @@ impl<'a> CrateMetadataRef<'a> {
         tcx.arena.alloc_from_iter(self.root.lang_items_missing.decode(self))
     }
 
+    fn get_exportable_items(self) -> impl Iterator<Item = DefId> + 'a {
```

Comment:
```suggestion
    fn get_exportable_items(self) -> impl Iterator<Item = DefId> {
```
IIUC, this should no longer be necessary on 2024 edition.
(Same in `get_stable_order_of_exportable_impls` below.)